deploy/docker/start.sh

```diff
@@ -37,8 +37,13 @@ tail -f /opt/emqx/log/erlang.log.1 &
 #          you must let user know emqx crashed and stop this container,
 #          and docker dispatching system can known and restart this container.
 IDLE_TIME=0
+ERTS=$(find /opt/emqx -name "erts*" -type d)
```

Comment:
`. /opt/emqx/releases/emqx_vars` to get `ERTS_PATH` exported.

Reply:
This does not work because RUNNER_ROOT_DIR points `/usr` when it is called by `start.sh`.

Reply:
ok.  `. /opt/emqx/release/emqx_vars`  should give you `REL_VSN`.
then there is no need to use `find` command